src/components/Widgets/SubscriptionsWidget.tsx

```diff
@@ -0,0 +1,143 @@
+            id="Path"
+          />
+          <path
+            d="M18.2795551,24.4374721 C18.0039398,24.4374721 17.7293568,24.332525 17.5192905,24.1224587 L14.0784007,20.6815689 C13.6584401,20.2616082 13.6584401,19.5811723 14.0784007,19.1610396 C14.4983613,18.740907 15.1787972,18.741079 15.5989299,19.1610396 L18.2795551,21.8416649 L24.4002099,15.7201498 C24.8201705,15.3001892 25.5023269,15.3001892 25.9224595,15.7201498 C26.3408717,16.1401104 26.3408717,16.8205464 25.9224595,17.240679 L19.0398197,24.1224587 C18.8297534,24.332525 18.5551704,24.4374721 18.2795551,24.4374721 Z"
+            id="Path"
+          />
+        </g>
+      </g>
+    </g>
+  </svg>
+);
+
+const cardData = {
+  active: {
+    title: 'Active',
+    variant: 'success'
+  },
+  expiringSoon: {
+    title: 'Expiring soon',
+    variant: 'warning'
+  },
+  expired: {
+    title: 'Expired',
+    variant: 'danger'
+  },
+  futureDated: {
+    title: 'Future dated',
+    customIcon: <OutlinedCalendarAltIcon />
+  }
+};
+
+const SubsWidget = () => {
+  const statusCardData = useStatus();
+  const isCardDataEmpty = useMemo(
+    () =>
+      !statusCardData.isLoading &&
+      statusCardData?.data &&
+      Object.keys(cardData).every((name: keyof typeof cardData) => statusCardData.data[name] === 0),
+    [statusCardData]
+  );
+
+  return (
+    <div className="subscription-inventory">
+      {isCardDataEmpty ? (
+        <EmptyState variant={EmptyStateVariant.large}>
+          <EmptyStateIcon icon={EmptyStateSubscriptionsIcon} />
+          <Title headingLevel="h4" size="lg">
+            No connected subscriptions
+          </Title>
+          <EmptyStateBody>
+            <Stack>
+              <StackItem>
+                We cannot find any Red Hat subscriptions attached to your account
+              </StackItem>
+            </Stack>
+          </EmptyStateBody>
+        </EmptyState>
+      ) : (
+        <Gallery
+          hasGutter
+          className="pf-v5-u-p-md"
+          style={{ display: 'flex', flexDirection: 'row' }}
+        >
+          {['active', 'expiringSoon', 'expired', 'futureDated'].map(
+            (name: keyof typeof cardData) => {
+              return (
+                <Link
+                  to="/subscriptions/inventory"
+                  style={{ flex: 1, textDecoration: 'none' }}
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  key={name}
+                >
+                  <Alert isInline {...cardData[name]}>
+                    {!statusCardData.isLoading && statusCardData?.data && (
+                      <p>{statusCardData.data[name]}</p>
+                    )}
```

Comment:
What do you think about having a loading spinner here while the data is loading?